### PR TITLE
New screening times

### PR DESCRIPTION
**Pull request by AnnaLenaO**
Route, component & script for screening times & updated page.jsx. 
The first screening time is automatically chosen when page load. 
If no available times exists a user message like "No available times" is shown instead. 
To handle the fact that times are dependent of dates there is a wrapped async await fetch for screening times. 
To handle error and user message also for dates that component is updated in the same way. 
This also keep a consistent approach how to structure the code like the rest of the project's booking part. 

The database for screenings has been filled with screenings of +2 hours. Therefore the time needs to be adjusted in the route for getting times. There is a comment of this in the route. 

The responsiveness for the red boxes containing dates, times & associated error messages are adjusted to fit the text. 

The database is at the moment filled with screenings until 2024-05-24 for the first 10 movies. 

Some functions might be mooved from both time and date component into scripts, like for example the sorting in ascending order. On the other hand there is not much code in each component and some specific code for these components. The overall readability and understanding of the code might be better if the code is not moved to separate script-files. 

Prettier has been used and also a commit hook. 

image
image

### CODE REVIEW

File: kinogoteborg/src/app/components/booking/screeningDates.jsx [hunk 1]
```diff
@@ -1,22 +1,29 @@
...
 
   useEffect(() => {
-    fetch(`http://localhost:3000/api/booking/${id}/dates`)
-      .then((res) => res.json())
-      .then((data) => {
+    const fetchedDataForDates = async () => {
+      const fetchedData = await fetchScreeningDates(id);
+
+      if (fetchedData.length === 0) {
+        const noScreeningDates = ["No available dates"];
```
Comment (MatsJoh50) [inline kinogoteborg/src/app/components/booking/screeningDates.jsx:14]: This seems reduntant?
cant you just use "setSvreeningDates("No  Available dates") on row 16?

Comment (AnnaLenaO) [reply kinogoteborg/src/app/components/booking/screeningDates.jsx:14]: No, that does not work because of row 57: 

{screeningDates.map((date, index) => (

Comment (MatsJoh50) [reply kinogoteborg/src/app/components/booking/screeningDates.jsx:14]: soooo, set....(["no dates"]) ?

**Review by MatsJoh50 — APPROVED** [review]
nice!

**Pull request merged**